### PR TITLE
Convert sys/test_dynflow to pytest

### PR DESCRIPTION
**Pull request by mshriver**
```
setup@localhost:~/repos/robottelo (pytest-dynflow-14000$%=) % pytest -v tests/foreman/sys/test_dynflow.py 
========================================================================================== test session starts ==========================================================================================
platform linux -- Python 3.8.8, pytest-6.2.2, py-1.10.0, pluggy-0.13.1 -- /home/setup/repos/robottelo/.robottelo/bin/python3.8
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/setup/repos/robottelo, configfile: pyproject.toml
plugins: forked-1.3.0, ibutsu-1.14.1, xdist-2.2.1, mock-3.5.1, reportportal-5.0.8, cov-2.11.1, services-2.2.1
collected 1 item                                                                                                                                                                                        

tests/foreman/sys/test_dynflow.py::test_positive_setup_dynflow PASSED                                                                                                                             [100%]

===================================================================================== 1 passed in 65.83s (0:01:05) ======================================================================================
```

### CODE REVIEW

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: tests/foreman/sys/test_dynflow.py

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: tests/foreman/sys/test_dynflow.py

File: tests/foreman/sys/test_dynflow.py [hunk 1]
```diff
@@ -19,30 +19,24 @@
...
+@pytest.mark.tier2
+def test_positive_setup_dynflow():
+    """Set dynflow parameters, restart it and check it adheres to them
+
+    :id: a5aaab5e-bc18-453e-a284-64aef752ec88
+
+    :expectedresults: Correct dynflow processes are running, respecting settings
+    """
+    commands = [
+        "cd /etc/foreman/dynflow/",
```
Comment (JacobCallahan) [inline tests/foreman/sys/test_dynflow.py:33]: Single quotes 🙃

**Review by JacobCallahan — CHANGES_REQUESTED** [review]

File: tests/foreman/sys/test_dynflow.py (hunk 1 above)
Comment (mshriver) [reply tests/foreman/sys/test_dynflow.py:33]: Leaving the double quotes in place that have single quotes in the bash command, as I don't want to screw with potential changes to bash command parsing. _Probably_ would be fine with double quotes but I don't think its worth making the change.

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: tests/foreman/sys/test_dynflow.py

**Review by omkarkhatavkar — APPROVED** [review]
ACK

File: tests/foreman/sys/test_dynflow.py (hunk 1 above)
Comment (JacobCallahan) [reply tests/foreman/sys/test_dynflow.py]: fair enough

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**